### PR TITLE
CC-761 - Show Page Layout Supports Embeded PDFS

### PR DESCRIPTION
**Pull request by raytiley**
This brings in `ember-pdf-js` but actually uses a custom component that behaves a bit better for scaling.

This alo applies a bunch of styling tweaks for the show page from Amber.

TODO - drop ember-pdf-js dependency and just rely of mozilla's pdf js directly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "ember-export-application-global": "^1.0.5",
     "ember-font-awesome": "3.0.4",
     "ember-load-initializers": "^0.6.3",
+    "ember-pdf-js": "0.2.0",
```
Comment (jzemke) [inline package.json]: do we still need this if it's dropped now?

File: app/controllers/show.js [hunk 2]
```diff
@@ -12,6 +13,25 @@ export default Ember.Controller.extend({
...
+    for (let i = 0; i < pdfDisplays.length; i++) {
+      let fd = pdfDisplays[i];
+      let fileField = this.get('model.show.customFields').find((field) => {
+        return field.type === 'file' && fd.get('showField') === field.showField;
+      });
+      if (fileField) {
+        let file = this.get('store').peekRecord('web-file', fileField.value);
+        if (/.+\.pdf$/.test(file.get('name'))) {
+          return {
+            url: 'http://localhost:4200/Cablecast/WebFiles/9635/Lance%20-%20Fill%20Gaps%20Feature%20Request.pdf', //Ember.get(file || {}, 'url'),
```
Comment (jzemke) [inline app/controllers/show.js]: does this need to be updated?

Comment (raytiley) [reply app/controllers/show.js]: good call...
need tot take that out... was just for testing.

File: package.json (hunk 1 above)
Comment (raytiley) [reply package.json]: we still need it... it does a bunch of the importing of the library and setup... were' just not using the actual supplied component.

**Pull request merged**